process/sync/baseSync.go

```diff
@@ -1123,6 +1127,31 @@ func (boot *baseBootstrap) waitForMiniBlocks() error {
 	}
 }
 
+func (boot *baseBootstrap) setAccountsStorerIdentifiers() error {
```

Comment:
Duplicated code here. Extract L1131-L1139 in a new func, and use it for both UserAccountsUnit and PeerAccountsUnit

Reply:
done